### PR TITLE
feat(pradeo): create parser

### PR DESCRIPTION
**Pull request by LeoArurault**
Creating Pradeo intake

### CODE REVIEW

**Review by squioc — DISMISSED** [review]
LGTM

File: Pradeo/pradeo/tests/match_threats.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "message": "Nov 15 08:51:42 subdomain.pradeo.net mtd-pradeosecuritysystems[6030]: {\"USER\":\"test_user\",\"SECTION\":\"MTD\\/Apps\",\"ACTION\":\"app_checking\",\"DESCRIPTION\":\"\",\"ITEM\":{\"APPLICATION_ID\":\"55936212\",\"APP_PKG_NAME\":\"com.an_app\",\"APP_VERSION\":\"4.394.10003\",\"APP_SHA1_SIG\":\"a92675ab3dafb37399c47a75ceac8effc4cb401d\"},\"ACTION_VALUES\":{\"ALLOWED\":\"true\",\"ACTION\":\"automatic\",\"POLICY\":\"Green\",\"MATCH_THREATS\":[\"cat_phone_cache_send\",\"cat_phone_device_info_send\",\"cat_phone_hardware_send\",\"cat_user_contact_info_send\",\"match_encrypt_with_key_downloaded_from_network\",\"match_exec_command_downloaded_from_network\",\"match_hide_app_icon_from_launcher\",\"match_priority\",\"match_rootkit\",\"match_rootkit_warning\",\"match_sms\"]}}\n"
+    },
+    "expected": {
+        "package": {
+            "checksum": "a92675ab3dafb37399c47a75ceac8effc4cb401d",
+            "name": "com.an_app",
+            "version": "4.394.10003"
+        },
+        "action": {
+            "name": "app_checking",
```
Comment (PierrickV) [inline Pradeo/pradeo/tests/match_threats.json:18]:
```suggestion
        "event": {
            "reason": "app_checking",
```

This looks more like `event.reason` to me.

File: Pradeo/pradeo/_meta/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+uuid: e30f7bcc-7c55-4666-9d32-61a0aa75a2c3
+name: pradeo
+slug: pradeo
+description: 'Pradeo protects organizations’ mobile devices, applications and data'
```
Comment (PierrickV) [inline Pradeo/pradeo/_meta/manifest.yml:4]:
```suggestion
description: Pradeo protects organizations’ mobile devices, applications and data
```

File: Pradeo/pradeo/_meta/fields.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+pradeo.allowed:
+  type: keyword
+  name: pradeo.allowed
+  description: Does policy allowed this app.
```
Comment (PierrickV) [inline Pradeo/pradeo/_meta/fields.yml:4]:
```suggestion
  description: Indicates if the application is allowed (green/orange) or denied (red)
```

Comment (PierrickV) [reply Pradeo/pradeo/_meta/fields.yml:4]: We should probably prefer descriptions from the official documentation

File: Pradeo/pradeo/_meta/fields.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+pradeo.policy:
+  type: keyword
+  name: pradeo.policy
+  description: The policy selected
+pradeo.match_threats:
+  type: keyword
+  name: pradeo.match_threats
+  description: The threats matched.
```
Comment (PierrickV) [inline Pradeo/pradeo/_meta/fields.yml:12]:
```suggestion
  description: List of items detected in the application with a moderate/high alert associated.
```

File: Pradeo/pradeo/tests/match_threats.json (hunk 1 above)
Comment (LeoArurault) [reply Pradeo/pradeo/tests/match_threats.json:18]: The value comes from `"ACTION"` field. I think it is more the name rather than the reason

**Review by PierrickV — APPROVED** [review]

**Pull request merged**